pkg/localcontroller/manager/dataset.go

```diff
@@ -105,6 +110,14 @@ func (dm *DatasetManager) Insert(message *gmclient.Message) error {
 		return err
 	}
 
+	credential := dataset.ObjectMeta.Annotations[Credential]
+	if credential != "" {
+		dataset.Storage = storage.Storage{}
+		if err := dataset.Storage.SetCredential(credential); err != nil {
+			klog.Errorf("dataset(name=%s) sets storage credential failed, error: %+v", name, err)
```

Comment:
nits: `Failed to set dataset(name=%s) sets storage credential failed, error:`

Reply:
fix it